File: apzkr-pzpi-21-5-vyshniak-viktoriia/Task1-Server/BLL/Infrastructure/Models/DiaryNote/DiaryNoteListItem.cs
namespace BLL.Infrastructure.Models.DiaryNote;
public class DiaryNoteListItem
{
    public Guid DiaryNoteId { get; set; }

    public Guid PetId { get; set; }

    public string Title { get; set; }

    public DateTime CreatedOnUtc { get; set; }

    public DateTime LastUpdatedOnUtc { get; set; }
}

File: apzkr-pzpi-21-5-vyshniak-viktoriia/Task4-MobileClient/app/src/main/java/com/pawpoint/model/response/DiaryNoteListItem.kt
package com.pawpoint.model.response

import com.google.gson.annotations.SerializedName

data class DiaryNoteListItem(

    @SerializedName("diaryNoteId")
    val diaryNoteId: String,

    @SerializedName("petId")
    val petId: String,

    @SerializedName("title")
    val title: String,

    @SerializedName("createdOnUtc")
    val createdOnUtc: String,

    @SerializedName("lastUpdatedOnUtc")
    val lastUpdateOnUtc: String
)
